package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rev-hash",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Because bumping the version is handled by a script for me. It's also up to the maintainer when to release and what version to release as. I might want to include other changes before doing a version bump.